dashboard/app/views/admin_reports/admin_stats.html.haml

```diff
@@ -1,88 +1,91 @@
 %h1 Admin stats
 
-%h3 Users with accounts
+-if @stats && @stats.length > 0
```

Comment:
you can use `@stats.present?` here http://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark